libs/react-spa/shared/src/lib/replaceParams.ts

```diff
@@ -0,0 +1,23 @@
+import { Params } from 'react-router-dom'
+
+type ReplaceParams = {
+  href: string
+  params: Params<string>
+  paramsPrefix?: string
+}
+
+/**
+ * Replace params in path
+ * @example
+ * replaceParams('/users/:id', { id: 1 }) // '/users/1'
+ */
+export const replaceParams = ({
+  href,
+  params,
+  paramsPrefix = '/',
```

Comment:
~The regex is replacing `/:paramName` and if this would be set to something else it would change this character so the url template `/users/:id` would be `/users+1` if the `paramsPrefix  = '+'`. But path params should always be separate path segments.

Is there a use case for this `paramsPrefix` that I'm not seeing or should we just remove it and have it fixed as `'/'`?

Reply:
Good point!! Will update